Makefile

```diff
@@ -100,6 +103,8 @@ test-unit: vendor
 	go test -v ./...
 
 # Functional tests assume access to github.com
+# and `brigade-github-app.enabled=true` set in the Brigade chart
```

Comment:
Should this be `--set brigade-github-app.enabled=true` ?

Reply:
Yes, that would be more explicit/helpful; I'll update.